k8s/namespaces/ia/ia-case-access-api/aat.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-access-api
+spec:
+  values:
+    java:
+      ingressHost: ia-case-access-api-aat.service.core-compute-demo.internal
```

Comment:
please check url, it should be `aat` instead of `demo`

Reply:
resolved